ext/nnstreamer/tensor_decoder/tensordec-directvideo.c

```diff
@@ -60,14 +95,31 @@ dv_exit (void **pdata)
 static int
 dv_setOption (void **pdata, int opNum, const char *param)
 {
-  /* We do not accept anything. */
+  direct_video_ops *ddata = *pdata;
+
+  /* When the dimension[0] is 4, the video format will be decided by option1. */
+  switch (opNum) {
+    case 0:
+      ddata->format = find_key_strv (dv_formats, param);
+
+      if (NULL == param || *param == '\0') {
```

Comment:
As `param` is const type, I think this statement should be moved to before `find_key_strv`

Reply:
that's right!